helm/values.yaml

```diff
@@ -0,0 +1,23 @@
+image:
+  init:
+    repository: monokle-admission-controller-init
+    pullPolicy: IfNotPresent
+    # Override the image tag whose default is the chart appVersion.
+    tag: ""
+    # Override entire image path.
+    overridePath: ""
+  server:
+    repository: monokle-admission-controller-server
+    pullPolicy: IfNotPresent
+    # Override the image tag whose default is the chart appVersion.
+    tag: ""
+    # Override entire image path.
+    overridePath: ""
+
+# Deploy Monokle Admission Controller resources to below namespace.
+namespace: monokle-admission-controller
+
+# Do not validate resources from below namespaces.
+ignoreNamespaces:
+#   - ns1
+#   - ns2
```

Comment:
Is this used tho? I see in deployment.yaml, it is not used for actual templating of the env variable.

Reply:
It is used in webhook manifest here - https://github.com/kubeshop/monokle-admission-controller/pull/8/files#diff-659c9537505abd4ab0fe12aa4a8747e1199cac92366649a827817ec5101ba317R26-R28